Makefile

```diff
@@ -229,7 +230,7 @@ generate-database-dumps:
 test-ci: export CI_BUILD=1
 test-ci:
 	CGO_ENABLED=$(CGO_ENABLED) BUILD_TAGS='$(BUILD_TAGS)' sh -c "'$(CURDIR)/scripts/build.sh'"
-	~/.go/bin/go test ./... -v $(TESTARGS) -timeout 120m
+	~/.go/bin/go test ./... -v $(TESTARGS) -json -timeout 120m | tparse -all
```

Comment:
Can we add this to the test target? It seems useful locally as well

Reply:
Sure!